memstore/recovery.go

```diff
@@ -228,13 +228,19 @@ func (m *memStoreImpl) InitShards(schedulerOff bool) {
 	if !schedulerOff {
 		// Start scheduler.
 		utils.GetLogger().Infof("Starting archiving scheduler")
+		// this will start scheduler of all jobs except archiving, archiving will be started individually
 		m.GetScheduler().Start()
 	} else {
 		utils.GetLogger().Infof("Scheduler is off")
 	}
 
 	m.replayRedoLogs()
 
+	if !schedulerOff {
```

Comment:
probably we don't need to check schedulerOff, since the scheduler loop is not running anyway